client/performance-profiler/components/core-web-vitals-display/index.tsx

```diff
@@ -8,30 +9,55 @@ import {
 import { CoreWebVitalsAccordion } from '../core-web-vitals-accordion';
 import { MetricTabBar } from '../metric-tab-bar';
 import { CoreWebVitalsDetails } from './core-web-vitals-details';
-
+import { CoreWebVitalsDetailsV2 } from './core-web-vitals-details_v2';
 import './style.scss';
 
 type CoreWebVitalsDisplayProps = Record< Metrics, number > & {
 	history: PerformanceMetricsHistory;
 	audits: Record< string, PerformanceMetricsItemQueryResponse >;
 	recommendationsRef: React.RefObject< HTMLDivElement > | null;
+	showV2?: boolean;
 };
 
+const MetricTabBarV2 = lazy( () => import( '../metric-tab-bar/metric-tab-bar-v2' ) );
```

Comment:
Because both versions of metric tab uses the same classnames (_this so can easily delete old css file when we fully switch to v2_). If I don't lazy load we have class names from both css files overriding each other and causing problems. So I lazy load for now.